### PR TITLE
Code style requests and limits for apt read

### PR DESCRIPTION
**Pull request by kalyanidani**
Readability changes for emphasis on **requests** and **limits** rather than resource type.

### CODE REVIEW

**Review by divya-mohan0209 — COMMENTED** [review]
I'm not sure what value add is there in this @kalyanidani . Could you please elaborate?

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
 However, a container is not allowed to use more than its resource `limit`.
 
-For example, if you set a `memory` request of 256 MiB for a container, and that container is in
+For example, if you set a memory `request` of 256 MiB for a container, and that container is in
```
Comment (chrisnegus) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:34]: The literal object set here `memory`, so I think the original is appropriate. The backticks are used to show literal values and the literal value here would actually be `requests`, as in `spec.containers[].resources.requests.memory`.

Comment (sftim) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:34]: The original is better. `memory` is literal value you would use in the API, but you would not put that in a field named `request`.

As you'll see from https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources the field is called `resources`, _plural_.

I do think the original text is better.

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
...
 a Pod scheduled to a Node with 8GiB of memory and no other Pods, then the container can try to use
 more RAM.
 
-If you set a `memory` limit of 4GiB for that container, the kubelet (and
+If you set a memory `limit` of 4GiB for that container, the kubelet (and
```
Comment (sftim) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:38]: The original is better. `memory` is a literal value you would use in the API, and you must type it exactly as written. As used here, “limit” is the English word “limit”.

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 3]
```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
 However, a container is not allowed to use more than its resource `limit`.
```
Comment (sftim) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:32]: Maybe this line is the one where a change would be helpful?

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 4]
```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
```
Comment (sftim) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:31]: Here, we could remove the backticks around `request`.

File: content/en/docs/concepts/configuration/manage-resources-containers.md (hunk 2 above)
Comment (chrisnegus) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:38]: Same issue here. In this case, `memory` is the value under `limits`, so keeping the original would be good or you could use the full `spec.containers[].resources.limits.memory`.

**Review by chrisnegus — COMMENTED** [review]
I have a couple of comments.

**Comment by reylejano** [comment]
I'm closing this PR since the original text is favored over the changes in this PR
/close

**Pull request closed without merging**